monolith/database.py

```diff
@@ -47,48 +48,54 @@ def get_id(self):
 class Story(db.Model):
     __tablename__ = 'story'
     id = db.Column(db.Integer, primary_key=True, autoincrement=True)
-    title= db.Column(db.Text(50))
-    text = db.Column(db.Text(1000)) # around 200 (English) words 
+    text = db.Column(db.Text(1000), nullable=True)  # around 200 (English) words
     date = db.Column(db.DateTime)
-    likes = db.Column(db.Integer) # will store the number of likes, periodically updated in background
-    dislikes = db.Column(db.Integer)
-    # define foreign key 
+    # will store the number of likes, periodically updated in background
+    likes = db.Column(db.Integer, default=0)
+    rolls_outcome = db.Column(db.Unicode(1000))
+    theme = db.Column(db.Unicode(128))
+    title = db.Column(db.Text(50), nullable=True)
     author_id = db.Column(db.Integer, db.ForeignKey('user.id'))
+    published = db.Column(db.Boolean, default=False)
+    # define foreign key
     author = relationship('User', foreign_keys='Story.author_id')
 
+    # TODO complete this method invocation about throw die button
     def __init__(self, *args, **kw):
         super(Story, self).__init__(*args, **kw)
         self.date = dt.datetime.now()
 
+    def setText(self, text):
+        self.text = text
+
+
 class Like(db.Model):
     __tablename__ = 'like'
-    
-    liker_id = db.Column(db.Integer, db.ForeignKey('user.id'), primary_key=True)
+
+    liker_id = db.Column(db.Integer, db.ForeignKey(
+        'user.id'), primary_key=True)
     liker = relationship('User', foreign_keys='Like.liker_id')
 
-    story_id = db.Column(db.Integer, db.ForeignKey('story.id'), primary_key=True)
+    story_id = db.Column(db.Integer, db.ForeignKey(
+        'story.id'), primary_key=True)
     author = relationship('Story', foreign_keys='Like.story_id')
 
-    marked = db.Column(db.Boolean, default = False) # True iff it has been counted in Story.likes
-    
-class Dislike(db.Model):
-    __tablename__ = 'dislike'
-    
-    disliker_id = db.Column(db.Integer, db.ForeignKey('user.id'), primary_key=True)
-    disliker = relationship('User', foreign_keys='Dislike.disliker_id')
+    liked_id = db.Column(db.Integer, db.ForeignKey(
+        'user.id'))  # TODO: duplicated ?
+    liker = relationship('User', foreign_keys='Like.liker_id')
```

Comment:
Old code should be used. Dislike support is being dropped for some reason.

Moreover the liker_id as been removed since it is superflous.